src/app/components/queue/CreateQuestion.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box, Button } from "@mui/material";
```

Comment:
Oops, duplicated with my implementation [here](https://github.com/johnny-t06/officeminutesv2/pull/25/files#diff-b99b071e626569a6c49511173bb0aec380c0f8dbf6fcf4a14f4d8820110de56bR63-R82)

Reply:
May be we can remove `private/course/[courseId]/queue.tsx` and `CreateQuestion.tsx` from the PR, since it's not directly related to the task?

Reply:
Edit: I see what you're doing with `CreateQuestion.tsx`. We probably should use this instead of my implementation.